### PR TITLE
EC2 files

### PR DESCRIPTION
**Pull request by nakonn**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+variable "instance_name" {
+  type    = list(string)  
+  description = "A list of names for the instance(s)."
+}
+
+variable "root_block_device" {
+  description = "Customize details about the root block device of the instance"
+  type        = list(any)
```
Comment (Jiraiya106) [inline variables.tf]: try don't use "any"

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  description = "A list of security group IDs to associate with the instances."
+}
+
+variable "environment" {
+  type        = string
+  description = "The environment for which the infrastructure is intended (e.g., dev, test, prod)"
+}
+
+variable "key_pair" {
+  description = "Name of the key pair public."
```
Comment (Jiraiya106) [inline variables.tf:57]: sensitive

Comment (nakonn) [reply variables.tf:57]: fixed

File: variables.tf (hunk 1 above)
Comment (nakonn) [reply variables.tf]: fixed

**Comment by Jiraiya106** [comment]
Aprove

**Pull request merged**